warehouse/templates/email/added-as-collaborator.body.txt

```diff
@@ -1,6 +1,17 @@
-You have been added as {{ role }} to the PyPI project {{ project }} by {{ submitter }}.
+You have been added as a collaborator to the project {{ project }}
+by {{ submitter }}.
+
+{% if role == 'Owner' -%}
+You are now a project owner. This means that you may add other collaborators,
+upload releases for the project, delete files and releases, or delete the entire
+project.
+{%- elif role == 'Maintainer' -%}
+You are now a project maintainer. This means that you may upload releases for
+the project. You cannot add collaborators, delete files, delete releases, or
+delete the project.
+{%- endif %}
 
 Congratulations!
```

Comment:
I'm not sure if "Congratulations!" is always appropriate for this email.  WDYT @di?